NEO-TAT/tat_flutter
# Separate build flavors

## Description 
Using production related links and env variables is not suitable for development testing, in addition, it makes wrong log event to the production server, such as firebase.

Hence, for a better devops experience, we did this change. Now you can choose build using BETA or REAL.

The firebase configuration files has been allowed to git history at this moment.
In this pr, they has been put into correct place, follow by different flavors.

Also, packages has been  upgraded too.

## How to Verify? 
Use Android Studio, or VSCode.

Both AOS and iOS:
- Build with BETA, see if app can be normally inited, and check if there were any error log.
- Build with REAL, see if app can be normally inited, and check if there were any error log.

## Screenshots/GIF/Test Results (Optional)
